src/noisepy/seis/noise_module.py

```diff
@@ -25,8 +25,8 @@
 from scipy.fftpack import next_fast_len
 from scipy.signal import hilbert
 
+from noisepy.seis.correlate import ConfigParameters
 from noisepy.seis.datatypes import ChannelData, StackMethod
```

Comment:
I'm not sure why ConfigParameters was getting imported from S1..., but it should be from .datatypes. A good opportunity to clean this up :)

```suggestion
from .datatypes import ChannelData, StackMethod, ConfigParameters
```